lib/pkgcloud/openstack/storage/client/containers.js

```diff
@@ -101,10 +102,14 @@ exports.createContainer = function (options, callback) {
     container: containerName
   };
 
+  var headers = options.headers || {};
```

Comment:
This appears to allow passing arbitrary headers to createContainer? If so, that should be added in the documentation at https://github.com/pkgcloud/pkgcloud/pull/385/files#diff-c4a3a7c9ff3534d7a13bcb9345003bc3L92, as well as in the documentation.